test_tableslurp.sh

```diff
@@ -0,0 +1,159 @@
+#! /bin/sh
+
+# Script Name: test_tableslurp.sh
```

Comment:
Suggestion: add a description and a usage() function like test_tablesnap.sh has.

Reply:
Added.